README.md

```diff
@@ -4,35 +4,90 @@ NodeJS >=18.19.0
 
 ## Installation
 
-```
+Before you proceed to the local development steps, you should install the dependencies. npm is bundled with node - which you will also need for running this project.
+
+```bash
 npm install
 ```
 
 ## Local Development
 
-1. **Start the Development Server:** Use the command `npm run dev` to launch `npm run dev:frontend` and `netlify-cms` concurrently.
+**Start the Development Server:** Use the command `npm run dev` to launch `npm run dev:frontend` and `netlify-cms` concurrently.
+
+### Previewing the Website
+
+You can now navigate to [http://localhost:3000](http://localhost:3000) to see a local preview of the website including your changes. The port may change when 3000 is taken - to check the console for any hint of that.
+
+### Accessing the CMS
+
+1. **Access the CMS:** Navigate to [http://localhost:3000/admin/index.html](http://localhost:3000/admin/index.html) in your web browser to open the Netlify CMS interface.
 
-2. **Access the CMS:** Navigate to [http://localhost:8888/admin/index.html](http://localhost:8888/admin/index.html) in your web browser to open the Netlify CMS interface.
+2. **Login:** Click on the login button. You should be granted access immediately. If prompted for a URL, enter the address of the proxy server - this shouldn't usually happen though.
 
-3. **Login:** Click on the login button. You should be granted access immediately. If prompted for a URL, enter the address of the proxy server.
+### Additional Background Information on the Setup
 
-### Notes
-- The `npm run dev:frontend` command runs your application and integrates netlify under the `/admin/index.html` routes. 
+- The `npm run dev:frontend` command runs your application and integrates netlify under the `/admin/index.html` routes.
 - The CMS proxy server enables the use of the local strategy for content management, directly saving files to the local file system.
 
+## Short File Tree Description
+
+```txt
+.
+├── environmentVariables.ts.template // Template file to inject .env variables. Will be filed wit envsubst in the build step. See "Building Details" for more info.
+├── migration-from-bsaber.com // Scripts used to transfer MotW from the old page - can be ignored
+├── README.md
+├── src
+│   ├── app.html // Entry point for the application - contains placeholder for special tags such as <svelte:head> as well
+│   ├── app.scss // Global styles
+│   ├── collections // Actual data for the collection per their model definiton. One folder = One Collection
+│   ├── lib // Components and functions that can be accessed with the $lib path
+│   ├── maps.ts // Central point for enums, that are shared across multiple components
```

Comment:
Unless "enums" is short for "enumerables" in this case, I don't think this is correct. All objects in that file are maps, or `Record`s in TypeScript terms.